mllib/src/main/scala/org/apache/spark/ml/clustering/LDA.scala

```diff
@@ -622,11 +623,11 @@ object LocalLDAModel extends MLReadable[LocalLDAModel] {
       val vectorConverted = MLUtils.convertVectorColumnsToML(data, "docConcentration")
       val matrixConverted = MLUtils.convertMatrixColumnsToML(vectorConverted, "topicsMatrix")
       val Row(vocabSize: Int, topicsMatrix: Matrix, docConcentration: Vector,
-          topicConcentration: Double, gammaShape: Double) =
+          topicConcentration: Double, gammaShape: Double, seed: Long) =
```

Comment:
This will break backwards compatibility of ML persistence (when users try to load LDAModels saved using past versions of Spark).  Could you please test this manually by saving a LocalLDAModel using Spark 2.3 and loading it with a build of your PR?  You can fix this by checking for the Spark version (in the `metadata`) and loading the seed for Spark >= 2.4.